querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java

```diff
@@ -57,6 +57,11 @@ public void  Signature() throws NoSuchMethodException {
                 assertReturnType(Expressions.class.getMethod(type + "Template", Class.class, Template.class, ImmutableList.class));
             }
         }
+
+        // arrays
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, String.class));
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, Path.class, String.class));
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, PathMetadata.class));
```

Comment:
It's a test for the naming convention, the return type simple name is the capitalized method name.